db/schema.rb

```diff
@@ -728,4 +728,34 @@
   add_foreign_key "vita_partner_zip_codes", "vita_partners"
   add_foreign_key "vita_partners", "coalitions"
   add_foreign_key "vita_providers", "provider_scrapes", column: "last_scrape_id"
+
+  create_view "organization_capacities", sql_definition: <<-SQL
+      WITH organization_id_by_vita_partner_id AS (
+           SELECT vita_partners_1.id,
```

Comment:
Do you need to rename the table as `vita_partners_1` rather than `vita_partners`? If not, it'd be nice to use the table name.